main.go

```diff
@@ -103,10 +105,17 @@ func main() {
 		Client:        mgr.GetClient(),
 		Scheme:        mgr.GetScheme(),
 		EventRecorder: mgr.GetEventRecorderFor(operatorName),
-	}).SetupWithManager(mgr, ratelimiter, flagVar.chartPath); err != nil {
+	}).SetupWithManager(mgr, ratelimiter); err != nil {
 		setupLog.Error(err, "unable to create controller", "controller", "Sample")
 		os.Exit(1)
 	}
+	if err = (&controllers.SampleHelmReconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
```

Comment:
No `EventRecorder` here causes panics in runtime - unable to send events (r.Event == nil)